pkg/segment/aggregations/segaggs.go

```diff
@@ -2408,6 +2414,665 @@ func performValueColRequest(nodeResult *structs.NodeResult, aggs *structs.QueryA
 	return nil
 }
 
+func getNumericValue(v interface{}) (float64, bool) {
+	switch num := v.(type) {
+	case int:
```

Comment:
If you just need float value, you can directly use this method:

```
ConvertToFloat(exp interface{}, bytes int)
```

in `dtypeutils.go`. And you can remove this method.

Reply:
addressed in the latest commit.